test/integration/pack/test_pack.py

```diff
@@ -1480,3 +1480,203 @@ def test_overwritten_version_file(cartridge_cmd, project_without_dependencies, t
+
+    rc, output = run_command_and_get_output(cmd, cwd=tmpdir)
+    assert rc == 0
+
+    archive_path = find_archive(tmpdir, project.name, pack_format)
+    extract_dir = os.path.join(tmpdir, 'extract')
+    extract_app_files(archive_path, pack_format, extract_dir)
+
+    for unit in files_by_units.keys():
+        filename = files_by_units[unit][0]
+        fd_limit = files_by_units[unit][1]
+        filepath = os.path.join(extract_dir, 'etc/systemd/system', filename)
+        with open(filepath) as f:
+            assert "LimitNOFILE={}".format(fd_limit) in f.read()
+
+
+@pytest.mark.parametrize('pack_format', ['rpm', 'deb'])
+def test_fd_limit_specified_with_flag(cartridge_cmd, project_without_dependencies, pack_format, tmpdir):
+    project = project_without_dependencies
+
+    fd_limit = 1024
+    stateboard_fd_limit = 2048
+
+    files_by_units = {
+        'unit': ["%s.service" % project.name, fd_limit],
+        'instantiated-unit': ["%s@.service" % project.name, fd_limit],
+        'stateboard-unit': ["%s-stateboard.service" % project.name, stateboard_fd_limit],
+    }
+
+    systemd_unit_params = os.path.join(tmpdir, "not-default-systemd-unit-params.yml")
+    with open(systemd_unit_params, "w") as f:
+        f.write(f"""
+                fd-limit: {fd_limit}
+                stateboard-fd-limit: {stateboard_fd_limit}
+                """)
+
+    cmd = [
+        cartridge_cmd,
+        "pack", pack_format,
+        "--unit-params-file", systemd_unit_params,
+        project.path,
+    ]
+
+    if platform.system() == 'Darwin':
+        cmd.append('--use-docker')
+
+    rc, output = run_command_and_get_output(cmd, cwd=tmpdir)
+    assert rc == 0
+
+    archive_path = find_archive(tmpdir, project.name, pack_format)
+    extract_dir = os.path.join(tmpdir, 'extract')
+    extract_app_files(archive_path, pack_format, extract_dir)
+
+    for unit in files_by_units.keys():
+        filename = files_by_units[unit][0]
+        fd_limit = files_by_units[unit][1]
+        filepath = os.path.join(extract_dir, 'etc/systemd/system', filename)
+        with open(filepath) as f:
+            assert "LimitNOFILE={}".format(fd_limit) in f.read()
+
+
+@pytest.mark.parametrize('pack_format', ['rpm', 'deb'])
+def test_fd_limit_default_values(cartridge_cmd, project_without_dependencies, pack_format, tmpdir):
+    project = project_without_dependencies
+
+    default_fd_limit = 65535
+    default_stateboard_fd_limit = 65535
+
+    files_by_units = {
+        'unit': ["%s.service" % project.name, default_fd_limit],
+        'instantiated-unit': ["%s@.service" % project.name, default_fd_limit],
+        'stateboard-unit': ["%s-stateboard.service" % project.name, default_stateboard_fd_limit],
+    }
+
+    cmd = [
+        cartridge_cmd,
+        "pack", pack_format,
+        project.path,
+    ]
+
+    if platform.system() == 'Darwin':
+        cmd.append('--use-docker')
+
+    rc, output = run_command_and_get_output(cmd, cwd=tmpdir)
```

Comment:
No output is needed here (use function without output or ``rc, _ = run_...``)

Reply:
`subprocess.run`?